splinepy/io/default.py

```diff
@@ -0,0 +1,71 @@
+import pathlib
+
+from splinepy.io import gismo, iges, irit, json, mfem, npz, vtk
+
+extension_to_io = {
+    ".g2": gismo,
```

Comment:
great guess, but gismo uses `.xml`.
This is sadly causes conflict between gismo and cats. @jzwar would it possible to identify them?

Reply:
it's a good time to consider #257

Reply:
> great guess, but gismo uses `.xml`.
> This is sadly causes conflict between gismo and cats. @jzwar would it possible to identify them?

Loading can differentiate using keywords. But it would be better to default XML to gismo as CATS is only used in very specific context

Reply:
Also: `xml` is not listed here. Maybe we can also wait with this for #297

Reply:
gismo has now `xml` extension
and vtk is removed.